tests/test_multi_segment.py

```diff
@@ -36,6 +36,7 @@ def test_linear_boundary_stages(config, filter_probs_df):
     assert_results(comp_stage.get_results())
 
 
+@pytest.mark.skip(reason="Skipping due to MRC issue #360")
```

Comment:
Is this something we can `xfail` instead? That way we can notified when it starts passing, which it would do if we fix the bug in MRC, but otherwise not notice it was fixed in Morpheus.

Reply:
In this case we cannot as marking a test `xfail` still executes the test, and this test would abort the current process.

Reply:
@dagardner-nv Can we remove this once we are using the fix in MRC?